### ISSUE TITLE
Spanish Translations

### ISSUE THREAD

**Issue by sailenicolas** [webyog/sqlyog-community#2169]
Is there any way to translate to Spanish? Is there a procedure? Or just editing files in
> sqlyog-community/localization/Strings

Is enough?

I am translator, And I wanted to translate it to Spanish, that is why I am asking.

**Comment by peterwy** [comment]
Yes, please refer this blog http://blog.sqlyog.com/how-to-localize-sqlyog/

However the resources for localization is now on github at https://github.com/webyog/sqlyog-community/tree/master/localization (and not googlecode as was the case when the blog was written).

Translating SQLyog to a new language is a major undertaking (what you might realize already).

**Comment by peterwy** [comment]
.. so there are 3 steps basically.

1) create a new Spanish language file with translated strings (and remember to manage the Windows API markup system for keyboard shortcuts. For instance "save" is possibly not (probably not) "Alt+s" in Spanish)
2) compile the new file into the l10n.db file using the "compile.bat" utility
3) replace l10n.db on the system(s) where SQLyog is installed with the new file you just created.

**Comment by gnovaro** [comment]
Cree un fork del repositorio y una rama nueva para el español. Les envié invitaciones para participar

**Comment by gnovaro** [comment]
I want to compile the application in Visual Studio Community 2019 but is so hard. And I have a few problems. For example the app have and external requirements libraries and not have and install instructions or listed in an file for example.

For example
Crypto++
https://www.cryptopp.com

Or
TintyXML2
https://github.com/leethomason/tinyxml2
